### PR TITLE
DOCS-5472: DBM AlwaysOn view

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do?
Adds a new guide for the DBM SQL Server AlwaysOn Clusters view.

### Motivation
DOCS-5472

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/guide/sql_alwayson.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+To identify groups that are experiencing issues, use the status filters to show groups with nodes that are Reverting, Not Synchronizing, and so forth. You can also use the timeseries graphs to spot unusual performance activity in your clusters based on log, redo, and secondary lag time metrics.
+
+{{< img src="database_monitoring/dbm_alwayson.png" alt="View SQL Server AlwaysOn groups" style="width:100%;">}}
+
+## Analyze historical metrics
+
+To evaluate how node synchronization states have fluctuated over time, select an availability group to open the details side panel. The **Historical Synchronization States** graph at the top of the panel shows the status of each node over the selected timeframe.
+
+View additional information about each secondary node on the **Secondary Nodes** tab. You can also use the timeseries graphs on the **Metrics** tab to spot abnormal behavior in individual nodes based on view send, redo, and lag metrics.
```
Comment (caseycull) [inline content/en/database_monitoring/guide/sql_alwayson.md:36]:
```suggestion
View additional information about each secondary node and its associated databases on the **Secondary Nodes** tab. You can also use the timeseries graphs on the **Metrics** tab to spot abnormal behavior in individual nodes and databases based on view send, redo, and lag metrics. 
```

**Review by caseycull — CHANGES_REQUESTED** [review]

**Comment by drichards-87** [comment]
Created DOCS-563 for the editorial review.

File: content/en/database_monitoring/guide/heroku-postgres.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ further_reading:
   text: "Datadog Heroku Buildpack"
 ---
 
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
```
Comment (hestonhoffman) [inline content/en/database_monitoring/guide/heroku-postgres.md:12]:
```suggestion
<div class="alert alert-warning">Database Monitoring is not supported on {{< region-param key=dd_datacenter >}}.</div>
```
Just a small QOL suggestion

File: content/en/database_monitoring/guide/sql_alwayson.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- link: "/database_monitoring/setup_sql_server/"
+  tag: "Documentation"
+  text: "Setting Up SQL Server"
+- link: "/database_monitoring/troubleshooting/"
+  tag: "Documentation"
+  text: "Troubleshooting Database Monitoring"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
```
Comment (hestonhoffman) [inline content/en/database_monitoring/guide/sql_alwayson.md:17]:
```suggestion
<div class="alert alert-warning">Database Monitoring is not supported on {{< region-param key=dd_datacenter >}}.</div>
```
As above

File: content/en/database_monitoring/guide/sql_alwayson.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+The Database Monitoring AlwaysOn Clusters view enables you to detect data synchronization issues, understand availability group behavior, and identify cluster bottlenecks in SQL Server availability groups.
+
+To access the AlwaysOn Clusters view, navigate to the **APM** > **Database Monitoring** > **Databases** tab and select **AlwaysOn Clusters**.
+
+## Determine the health of your nodes
+
+Use the AlwaysOn Clusters view to evaluate the health of your SQL Server availability groups. When selected, the page shows a color-coded visualization based on the current status of the primary (P) and secondary (S) nodes in each availability group.
+
+To identify groups that are experiencing issues, use the status filters to show groups with nodes that are Reverting, Not Synchronizing, and so forth. You can also use the timeseries graphs to spot unusual performance activity in your clusters based on log, redo, and secondary lag time metrics.
```
Comment (hestonhoffman) [inline content/en/database_monitoring/guide/sql_alwayson.md:28]: I can't find any guidance on this in our style guides, but should these UI terms (Reverting, Not Synchronizing) be italicized/bold? I think I'd usually bold them, but it's up to you.

**Review by hestonhoffman — APPROVED** [review]
Left you some suggestions, but LGTM!

**Pull request merged**